src/Http/Controllers/HandlesUploadedFilesInRequest.php

```diff
@@ -17,31 +19,60 @@ protected function handlePostedFiles(
         $instanceId = null
     ): void {
         collect($form->fieldsContainer()->getFields())
-            ->filter(fn ($field) => $field instanceof IsUploadField)
-            ->each(function (IsUploadField $field) use ($instanceId, $request, $formattedData) {
-                $wasUploaded = ($request[$field->key]['uploaded'] ?? false)
-                    && ($formattedData[$field->key]['file_name'] ?? false);
-                $wasTransformed = $field->isTransformOriginal()
-                    && ($request[$field->key]['transformed'] ?? false);
-
-                if ($wasUploaded) {
-                    HandleUploadedFileJob::dispatch(
-                        uploadedFileName: $request[$field->key]['name'],
+            ->each(function (SharpFormField $field) use ($instanceId, $request, $formattedData) {
+                if ($field instanceof SharpFormUploadField) {
+                    $this->handleFieldPostedFile(
+                        fileData: $request[$field->key] ?? null,
                         disk: $field->storageDisk(),
-                        filePath: $formattedData[$field->key]['file_name'],
-                        instanceId: $instanceId,
+                        filePath: $formattedData[$field->key]['file_name'] ?? null,
+                        transformOriginal: $field->isTransformOriginal(),
                         shouldOptimizeImage: $field->isShouldOptimizeImage(),
-                        transformFilters: $wasTransformed
-                            ? $request[$field->key]['filters']
-                            : null,
-                    );
-                } elseif ($wasTransformed) {
-                    HandleTransformedFileJob::dispatch(
-                        disk: $field->storageDisk(),
-                        fileData: $formattedData[$field->key]['file_name'],
-                        transformFilters: $request[$field->key]['filters'],
+                        instanceId: $instanceId
                     );
+                } elseif ($field instanceof SharpFormEditorField) {
+                    collect($request[$field->key]['files'] ?? [])
+                        ->each(function (array $file) use ($instanceId) {
+                            $this->handleFieldPostedFile(
+                                fileData: $file,
+                                disk: $file['disk'] ?? 'local',
+                                filePath: $file['path'] ?? null,
+                                transformOriginal: $file['transformOriginal'] ?? false,
+                                shouldOptimizeImage: $file['shouldOptimizeImage'] ?? false,
+                                instanceId: $instanceId,
+                            );
+                        });
```

Comment:
@dvlpp I handle editor files here, I have factorized in a `handleFieldPostedFile` method